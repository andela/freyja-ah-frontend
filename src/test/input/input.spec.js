import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import InputField from '../../components/inputs/input';

const setUp = (props = {}) => {
  const component = shallow(
    <InputField
      {...props}
      type="text"
      placeholder="Last name"
      name="lastName"
    />,
  );
  return component;
};
describe('InputField component', () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  it('should render without errors', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
