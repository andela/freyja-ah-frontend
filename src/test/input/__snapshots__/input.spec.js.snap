// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputField component should render without errors 1`] = `
<Fragment>
  <FormGroup
    className="inf"
    tag="div"
  >
    <Input
      className="input firstname is-invalid"
      name="lastName"
      placeholder="Last name"
      type="text"
    />
  </FormGroup>
</Fragment>
`;
